src/vmm/src/guest_config/aarch64_templates/mod.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::BTreeMap;
+
+use arch::aarch64::regs::Error;
+use kvm_ioctls::VcpuFd;
+
+/// Module with static aarch64 template
+pub mod g3g2;
+
+/// Structure that represents change in the register
+/// We zero bits with zero mask and set with values mask
+/// (register & mask) | value
```

Comment:
I thought that if a bit of mask is 1 use cpu template, otherwise use host value. In other words, `(value & mask) | (register & !mask)`. Please correct me if i'm wrong.

Reply:
Dicussed offline, we need more inputs for this part. So we'll leave as it is as of now and fix as follow-up if neded.